Storage/Storage/SQL/SQLiteHistory.swift

```diff
@@ -35,10 +36,32 @@ public class SQLiteHistory : History {
         }
     }
 
+    public class WrappedCursor : Cursor {
+        private let cursor: Cursor
+        override public var count: Int {
+            get { return cursor.count }
+        }
+
+        init(cursor: Cursor) {
+            self.cursor = cursor
+            super.init(status: cursor.status, msg: cursor.statusMessage)
+        }
+
+        // Collection iteration and access functions
+        public override subscript(index: Int) -> Any? {
+            get {
+                if let (site, visit) = cursor[index] as? (Site, Visit) {
```

Comment:
If you don't care about `visit` then you can do this: `if let (site, _) = cursor[index] as? (Site, Visit)`.